### ISSUE TITLE
Missing type exports

### ISSUE THREAD

**Issue by Haschtl** [jdalrymple/gitbeaker#3251]
**Description**

- Node.js version: 19
- Gitbeaker version: 38.3
- Gitbeaker release (cli, node, browser, core, requester-utils): core
- OS & version: Windows 11

Some types are not exported in the npm package. At least `MilestoneSchema` and `LabelSchema`.
Also `ExpandedProjectSchema` has missing `tag_list`.

```
Module '"@gitbeaker/core"' declares 'MilestoneSchema' locally, but it is not exported.ts(2459)
index.d.ts(2697, 11): 'MilestoneSchema' is declared here.
```

I also checked the `index.d.ts` directly on [npmjs.com](https://www.npmjs.com/package/@gitbeaker/core?activeTab=code), they are not beeing exported.

**Checklist**
- [ x] I have checked that this is not a duplicate issue.
- [ x] I have read the documentation.

**Comment by jdalrymple** [comment]
Will fix. tag_list is [deprecated](https://docs.gitlab.com/15.9/ee/api/projects.html#get-single-project) though.

**Comment by jdalrymple** [comment]
Release is deplayed. There seems to be a change affecting the accepted encoding thats crashing tests.

**Comment by jdalrymple** [comment]
darn scratch that. seems related to the cors thing again. Follow up when i know more

**Comment by jdalrymple** [comment]
haha asking for help to figure it out: https://stackoverflow.com/questions/76236958/how-to-use-nodejs-browsers-native-fetch-library-to-download-a-gitlab-archive-wit

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`38.4.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/38.4.0) :rocket:
